descriptors/ModuleDescriptor-template.json

```diff
@@ -307,7 +307,8 @@
             "inventory-storage.items.item.get",
             "inventory-storage.items.collection.get",
             "inventory-storage.items.item.put",
-            "users.item.get"
+            "users.item.get",
+            "user-tenants.collection.get"
```

Comment:
this is used by orders-storage Item Update Kafka consumer to update releted piece(that can be located in another tenant)